packages/@lwc/babel-plugin-component/src/decorators/wire/transform.js

```diff
@@ -63,6 +167,14 @@ function buildWireConfigValue(t, wiredValues) {
                 wireConfig.push(t.objectProperty(t.identifier('method'), t.numericLiteral(1)));
             }
 
+            wireConfig.push(
+                t.objectProperty(
+                    t.identifier('hasParams'),
```

Comment:
Do we really need an extra parameter to know if the wire config has dynamic params?

Reply:
@pmdartus with this reform there is no need of `params` or `static` in the wire meta. it hasn't being removed because of the probable impact in the platform metadata compiler.

once this pr is merged we will proceed to update the platform metadata compiler and remove the `params` and `static` meta from the wire definition.